.github/CONTRIBUTING.md

```diff
@@ -777,6 +777,42 @@ Isn't that confusing?
 
 There is also an undocumented keyword called `static` that has the same behaviour as global but more correctly describes BYOND's behaviour. Therefore, we always use static instead of global where we need it, as it reduces suprise when reading BYOND code.
 
+### Byond Hellspawn 
+
+Put stuff that shouldn’t work but does, or should work but doesn’t here so we don’t forget about it.
+#### \
+
+\ tells the compiler to read the next line as if it was one of our own. As an example:
```

Comment:
do we need to put this in contributing.md? dreamchecker picks it up

Reply:
ah we have a lint for this now? Great!

Reply:
ya

Reply:
haha yeah but who could benefit from that I wouldn't know them 😳